sample_logs/graphdb/create_graph.cypher

```diff
@@ -0,0 +1,98 @@
+CREATE (t9:Transaction {transaction_id: 9, transaction_date: '2024-01-09', narrative: 'Groceries', type: 'Debit', amount: 200, balance: -1410})
+CREATE (t10:Transaction {transaction_id: 10, transaction_date: '2024-02-10', narrative: 'Bills', type: 'Debit', amount: 800, balance: -2210})
+CREATE (t11:Transaction {transaction_id: 11, transaction_date: '2024-02-11', narrative: 'Bills', type: 'Debit', amount: 2500, balance: -4710})
+CREATE (t12:Transaction {transaction_id: 12, transaction_date: '2024-01-12', narrative: 'Entertainment', type: 'Debit', amount: 20, balance: -4730})
+CREATE (t13:Transaction {transaction_id: 13, transaction_date: '2024-01-13', narrative: 'Shopping', type: 'Debit', amount: 600, balance: -5330})
+CREATE (t14:Transaction {transaction_id: 14, transaction_date: '2024-01-14', narrative: 'Groceries', type: 'Debit', amount: 50, balance: -5380})
+CREATE (t15:Transaction {transaction_id: 15, transaction_date: '2024-01-15', narrative: 'Entertainment', type: 'Debit', amount: 15, balance: -5395})
+
+CREATE (p1)-[:HAS]->(a1)
+CREATE (p2)-[:HAS]->(a2)
+CREATE (p3)-[:HAS]->(a3)
+
+CREATE (p1)-[:MADE]->(t1)
+CREATE (p1)-[:MADE]->(t2)
+CREATE (p2)-[:MADE]->(t3)
+CREATE (p3)-[:MADE]->(t4)
+CREATE (p1)-[:MADE]->(t5)
+CREATE (p1)-[:MADE]->(t6)
+CREATE (p1)-[:MADE]->(t7)
+CREATE (p1)-[:MADE]->(t8)
+CREATE (p2)-[:MADE]->(t9)
+CREATE (p2)-[:MADE]->(t10)
+CREATE (p3)-[:MADE]->(t11)
+CREATE (p1)-[:MADE]->(t12)
+CREATE (p2)-[:MADE]->(t13)
+CREATE (p3)-[:MADE]->(t14)
+CREATE (p1)-[:MADE]->(t15)
+
+CREATE (t1)-[:PAID_TO]->(m1)
+CREATE (t2)-[:PAID_TO]->(m2)
+CREATE (t3)-[:PAID_TO]->(m2)
+CREATE (t4)-[:PAID_TO]->(m3)
+CREATE (t5)-[:PAID_TO]->(m1)
+CREATE (t6)-[:PAID_TO]->(m2)
+CREATE (t7)-[:PAID_TO]->(m3)
+CREATE (t8)-[:PAID_TO]->(m2)
+CREATE (t9)-[:PAID_TO]->(m1)
+CREATE (t10)-[:PAID_TO]->(m3)
+CREATE (t11)-[:PAID_TO]->(m2)
+CREATE (t12)-[:PAID_TO]->(m3)
+CREATE (t13)-[:PAID_TO]->(m1)
+CREATE (t14)-[:PAID_TO]->(m2)
+CREATE (t15)-[:PAID_TO]->(m3)
+
+CREATE (t1)-[:CLASSIFIED_AS]->(c1)
+CREATE (t2)-[:CLASSIFIED_AS]->(c2)
+CREATE (t3)-[:CLASSIFIED_AS]->(c2)
+CREATE (t4)-[:CLASSIFIED_AS]->(c2)
+CREATE (t5)-[:CLASSIFIED_AS]->(c1)
+CREATE (t6)-[:CLASSIFIED_AS]->(c2)
+CREATE (t7)-[:CLASSIFIED_AS]->(c2)
+CREATE (t8)-[:CLASSIFIED_AS]->(c2)
+CREATE (t9)-[:CLASSIFIED_AS]->(c1)
+CREATE (t10)-[:CLASSIFIED_AS]->(c2)
+CREATE (t11)-[:CLASSIFIED_AS]->(c2)
+CREATE (t12)-[:CLASSIFIED_AS]->(c2)
+CREATE (t13)-[:CLASSIFIED_AS]->(c1)
+CREATE (t14)-[:CLASSIFIED_AS]->(c2)
+CREATE (t15)-[:CLASSIFIED_AS]->(c2)
+
+CREATE (a1)<-[:PAID_BY]-(t1)
+CREATE (a1)<-[:PAID_BY]-(t2)
+CREATE (a2)<-[:PAID_BY]-(t3)
+CREATE (a3)<-[:PAID_BY]-(t4)
+CREATE (a1)<-[:PAID_BY]-(t5)
+CREATE (a1)<-[:PAID_BY]-(t6)
+CREATE (a1)<-[:PAID_BY]-(t7)
+CREATE (a1)<-[:PAID_BY]-(t8)
+CREATE (a2)<-[:PAID_BY]-(t9)
+CREATE (a2)<-[:PAID_BY]-(t10)
+CREATE (a3)<-[:PAID_BY]-(t11)
+CREATE (a1)<-[:PAID_BY]-(t12)
+CREATE (a2)<-[:PAID_BY]-(t13)
+CREATE (a3)<-[:PAID_BY]-(t14)
+CREATE (a1)<-[:PAID_BY]-(t15)
```

Comment:
GitHub is complaining about the lack of a newline at the end of this file. If we're making any changes, let's fix this too.